scripts/influxd-systemd-start.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+/usr/bin/influxd -config /etc/influxdb/influxdb.conf $INFLUXD_OPTS &
+echo $! > /var/lib/influxdb/influxd.pid
+
+BIND_ADDRESS=$(influxd print-config --key-name http-bind-address)
+HOST=${BIND_ADDRESS%%:*}
+HOST=${HOST:-"localhost"}
+PORT=${BIND_ADDRESS##*:}
+
+set +e
+result=$(curl -s -o /dev/null http://$HOST:$PORT/ready -w %{http_code})
```

Comment:
nitpick: this might have been cleaner:

```
result=
while [ "$result" != "200" ]; do
   result=$(curl -s -o /dev/null http://$HOST:$PORT/ready -w %{http_code})
   sleep 1
 done
```